src/main.rs

```diff
@@ -126,20 +134,36 @@ impl<'a> Printer<'a> {
     }
 
     fn header(&mut self) {
-        writeln!(
-            self.stdout,
-            "┌{0:─<8}┬{0:─<25}┬{0:─<25}┬{0:─<8}┬{0:─<8}┐",
-            ""
-        )
+        match self.border_style {
+            BorderStyle::Unicode => writeln!(
+                self.stdout,
+                "┌{0:─<8}┬{0:─<25}┬{0:─<25}┬{0:─<8}┬{0:─<8}┐",
+                ""
+            ),
+            BorderStyle::Ascii => writeln!(
+                self.stdout,
+                "+{0:-<8}+{0:-<25}+{0:-<25}+{0:-<8}+{0:-<8}+",
+                ""
+            ),
+            _ => Ok(())
```

Comment:
Could you replace `_` here and at other places to `BorderStyle::None`? That will make it harder to forget to change one match when a new border style is introduced.

Reply:
Sure, will do.

Reply:
Done.